content/tbb/export-and-import-bookmarks/contents.lr

```diff
@@ -0,0 +1,51 @@
+_model: question
+---
+title: How can I Export and Import bookmarks in Tor Browser?
+---
+seo_slug: export-and-import-bookmarks
+---
+description:
+
+Bookmarks in the Tor Browser can be exported, imported, backup, restored as well as imported from another browser.
+In order to export, import, backup or restore bookmarks in Tor Browser, go to:
+
+- Hamburger menu >> Library >> Bookmarks >> Show All Bookmarks (below the menu)
+- From the toolbar on the Library window, click Import and Backup
+
+**If you wish to export bookmarks**
+- Choose Export Bookmarks to HTML
+- In the Export Bookmarks File window that opens, choose a location to save the file, which is named bookmarks.html by default. The desktop is usually a good spot, but any place that is easy to remember will work.
+- Click the Save button. The Export Bookmarks File window will close.
+- Close the Library window.
+
+> Your bookmarks are now successfully exported from Tor Browser. The bookmarks HTML file you saved is now ready to be imported into another web browser.
+
+**If you wish to Import bookmarks**
+- Choose Import Bookmarks from HTML
+- Within the Import Bookmarks File window that opens, navigate to the bookmarks HTML file you are importing and select the file.
+- Click the Open button. The Import Bookmarks File window will close.
+- Close the Library window.
+
+> The bookmarks in the selected HTML file will be added to your Tor Browser within the Bookmarks Menu directory.
+
+**If you wish to Backup**
+- Choose Backup
+- A new window opens and you have to choose the location to save the file. The file has a .json extension.
+
+**If you wish to Restore**
+- Choose Restore and then select the bookmark file you wish to restore. 
+- Click okay to the pop up box that appears and hurray, you just restored your backup bookmark.
+
+**Import data from another browser**
+> Bookmarks can be transferred from Firefox to Tor Browser. There are two ways to export and import bookmarks in Firefox: [HTML file](https://support.mozilla.org/en-US/kb/export-firefox-bookmarks-to-backup-or-transfer) or JSON file. Remember any custom bridges and search engine additions.
+After exporting the data from the browser, follow the above step to import the bookmark file into your Tor Browser.
+
+**Export and Import bookmarks from Tor Browser for Android**
```

Comment:
@renateEta if you didn't run and try these instructions, I'd recommend removing Export and Import bookmarks from Tor Browser for Android part, so we can merge the rest.